src/components/01-branding/02-typography/fields.njk

```diff
@@ -1,69 +1,67 @@
 <h1>Fields</h1>
-<p>Examples of how to display fields.</p>
-
 
 <h2>Inline Field</h2>
 <p>Display name and value inline.</p>
 <div class="grid-row">
     <div class="sds-field">
-        <div class="sds-name">Field:</div>
-        <div class="sds-value">Lorem ipsum dolor</div>
+        <div class="sds-field--name">Field:</div>
+        <div class="sds-field--value">Lorem ipsum dolor</div>
```

Comment:
For BEM naming, this is my take:
**Block**: sds-field
**Modifier (variant)**: sds-field--small, sds-field--featured ... (along with all the variants)
**Element (child)**: sds-field__name
**Element (child)**: sds-field__value